installer/templates/phx_single/config/runtime.exs

```diff
@@ -19,7 +19,11 @@ if config_env() == :prod do
       You can generate one by calling: mix phx.gen.secret
       """
 
+  host = System.get_env("URL_HOST") || "example.com"
+  server? = System.get_env("SERVER") == "true"
+
   config :<%= @app_name %>, <%= @endpoint_module %>,
+    url: [host: host, port: 80],
```

Comment:
Not sure if I am happy with the hardcoded port here. I am wondering if instead we could support `url` as a string which we will parse. So we could do: `url: System.get_env("PHX_URL")`, and we set `PHX_URL=example.com:80`. The code would parse the string and discard all nil fields?